FlagsmithClient/src/main/java/com/flagsmith/entities/IdentityFlagsAndTraits.kt

```diff
@@ -2,5 +2,6 @@ package com.flagsmith.entities
 
 data class IdentityFlagsAndTraits(
     val flags: ArrayList<Flag>,
-    val traits: ArrayList<Trait>
+    val traits: ArrayList<Trait>,
+    val transient: Boolean = false
```

Comment:
This looks like the return data type, are we actually returning the `transient` boolean from the API? According to [this](https://github.com/Flagsmith/flagsmith/blob/main/api/environments/sdk/serializers.py#L138) (specifically `write_only=True`), I don't think that we are.

Reply:
Removed